consensus/ethash/sealer.go

```diff
@@ -243,7 +244,7 @@ func (ethash *Ethash) remote(notify []string, noverify bool) {
 		currentWork[1] = common.BytesToHash(SeedHash(block.NumberU64())).Hex()
 		currentWork[2] = common.BytesToHash(new(big.Int).Div(two256, block.Difficulty()).Bytes()).Hex()
 		currentWork[3] = hexutil.EncodeBig(block.Number())
-
```

Comment:
nit: I think it'd better to keep the blank line as before ;)

Reply:
Thanks for catching it!